src/flamegraph.js

```diff
@@ -443,6 +443,9 @@ export default function () {
             }).on('mouseout', function () {
                 if (tooltip) tooltip.hide()
                 detailsHandler(null)
+            }).on('mousemove', function (d) {
+                if (tooltip) tooltip.show(d, this)
+                detailsHandler(labelHandler(d))
```

Comment:
It is the same thing that is being called for 'mouseover'

Reply:
`mouseover` could be removed in this case

Reply:
yes, removed in latest commit
without mouseover also, it is working fine.